### DISCUSSION TITLE
i have a problem with setting plagin

### DISCUSSION THREAD

**Discussion by Blacksharks3** [JotaDF/moodle-blockgame#17] [Q&A] [answered]
hi, i really love your plugin and our team love it too. We are developing courses for schoolchildren and it is as if it was created for our courses. But we ran into one problem. when I go to the rating in the course, the message "It is necessary to make the initial settings of the Game Block in the course." We can't figure out what the problem is, I went into the plugin settings inside the course. I set up all the values ​​as I need, clicked on the save button. Points are awarded. But when you click on the rating button, an error occurs. tell me what to do?

![image](https://user-images.githubusercontent.com/108508403/197814146-6b956e97-80a7-4188-911b-83510b6b7fbb.png)
![image](https://user-images.githubusercontent.com/108508403/197814461-d383c83b-3fb1-4f77-a482-f72ed3d0027b.png)
![image](https://user-images.githubusercontent.com/108508403/197814473-66e4f00f-37f6-452c-a684-9cc6af20560a.png)
![image](https://user-images.githubusercontent.com/108508403/197814480-55abc119-ed41-4a04-b24d-d4aab5576110.png)
![image](https://user-images.githubusercontent.com/108508403/197814983-54b71756-871a-44b3-a60f-c8fd49005bab.png)

**Comment by jotadf** [comment, marked as answer]
Hello,
Could you send me a print of the last screen that contains the ranking error in English?

Thanks

**Reply by Blacksharks3** [reply]
"It is necessary to make the initial settings of the Game Block in the course."

**Reply by jotadf** [reply]
I simulated your configuration in my environment and it didn't show the error. Can you email me so we can try something I can help you with? j.wilson.df@gmail.com

**Reply by jotadf** [reply]
Apparently it is not saving the settings.
